static/app/views/replays/utils/indexReplayEntriesById.tsx

```diff
@@ -0,0 +1,8 @@
+import {ReplayDurationAndErrors} from '../replayTable';
```

Comment:
Ah I see why it's exported... I think we should get rid of this file and function and call `Object.fromEntries` where we need it.